notebooks/dc2/truth_variable_info.ipynb

```diff
@@ -0,0 +1,520 @@
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def flux2mag(flux):\n",
+    "    \"\"\"\n",
+    "    A convenience function to convert flux to magnitude\n",
+    "    \"\"\"\n",
+    "    with np.errstate(divide=\"ignore\"):\n",
+    "        return 22.5 - 2.5 * np.log10(flux)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# TODO: Remove this cell in final version\n",
+    "GCRCatalogs.set_root_dir(\"/global/cfs/cdirs/lsst/gsharing\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Supernova light curves\n",
+    "\n",
+    "We will start with finding light curves for supernovae.  When accessing DC2 variable truth information, one important concept is that the light curves are stored separetely from a \"summary table\". \n",
+    "\n",
+    "In the summary table, each row corresponds to one object (supernova), and it contains \"summary\" truth information such as coordinates, redshift, and time-averaged fluxes, but not the light curves. \n",
+    "\n",
+    "We can use the summary table to figure out the IDs of supernovae that we are interested in (e.g., in a certain sky area, in certain redshift ranges), these IDs will later be used to extract light curves.\n",
+    "\n",
+    "### Loading supernova summary table\n",
+    "\n",
+    "We first load the summary table into `sn_summary_cat` and check out what columns are available."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "sn_summary_cat = GCRCatalogs.load_catalog(\"desc_dc2_run2.2i_truth_sn_summary\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "sorted(sn_summary_cat.list_all_quantities())"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We will now actually load the data into memory. For this tutorial, we will only need the ID (`id`), coordinates (`ra`, `dec`), redshift. \n",
+    "\n",
+    "Recall that `get_quantities` returns a Python dictionary. We can use `pd.DataFrame` to turn it into a Pandas DataFrame."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "d_summary = pd.DataFrame(sn_summary_cat.get_quantities([\"id\", \"ra\", \"dec\", \"redshift\"]))\n",
+    "d_summary.head()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now that we have the summary table, we can use it to select, say supernovae near a certain coordinate or in a certain redshift range. In this tutorial, we will find all supernovae that are within 2 arcmin of (RA, Dec) = (60.5, -36.6) and within z < 0.8. We will store these supernovae in `objects_to_plot` which we will use later to extract and plot the their light curves."
```

Comment:
Suggest ending sentence with
"all supernovae that are within 2 arcmin of (RA, Dec) = (60.5, -36.6) and satisfy z < 0.8."